castai/types.go

```diff
@@ -186,7 +186,15 @@ func (c *ChartSource) Validate() error {
 	return nil
 }
 
+type NodeArchitecture int32
+
+const (
+	NodeArchitectureAmd64 NodeArchitecture = 0
+	NodeArchitectureArm64 NodeArchitecture = 1
```

Comment:
why not just sending `arm64/amd64` instead? 🤔

Reply:
I designed the api to not rely on strings, I think it is easier to transform and make operations.

Reply:
But it is much easier to reason about as 0/1 doesn't says much and you need to do additional mapping everywhere down the pipeline

Reply:
changed as suggested